src/main/java/com/internet/shop/dao/OrderDaoJdbcImpl.java

```diff
@@ -0,0 +1,178 @@
+            String query = "UPDATE orders "
+                    + "SET deleted = TRUE WHERE order_id = ?";
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setLong(1, id);
+            return preparedStatement.executeUpdate() != 0;
+        } catch (SQLException e) {
+            throw new DataBaseProcessingException("Order is not deleted", e);
+        }
+    }
+
+    private boolean deleteProductsInOrder(Long id) {
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            String query = "DELETE FROM orders_products WHERE order_id = ?";
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setLong(1, id);
+            return preparedStatement.executeUpdate() != 0;
+        } catch (SQLException e) {
+            throw new DataBaseProcessingException("Order product in order "
+                    + id + " was not deleted", e);
+        }
+    }
+
+    private Order addProductsIntoOrder(Order order) {
+        String query = "INSERT INTO orders_products (order_id, product_id) "
+                + "VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            for (Product product : order.getProducts()) {
+                preparedStatement.setLong(1, order.getId());
+                preparedStatement.setLong(2, product.getId());
+                preparedStatement.addBatch();
+            }
+            preparedStatement.executeBatch();
+            return order;
+        } catch (SQLException e) {
+            throw new DataBaseProcessingException("Failed to add the products to"
+                    + order, e);
+        }
+    }
+
+    private void getOrderProducts(Order order) {
+        List<Product> products = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            String query = "SELECT * FROM products JOIN orders_products "
+                    + "ON products.product_id = orders_products.product_id "
+                    + "WHERE orders_products.order_id = ?";
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setLong(1, order.getId());
+            ResultSet resultSet = preparedStatement.executeQuery();
+            while (resultSet.next()) {
+                Product product = new Product(
+                        resultSet.getString("name"),
+                        resultSet.getDouble("price"));
+                product.setId(resultSet.getLong("product_id"));
+                products.add(product);
+            }
+        } catch (SQLException e) {
+            throw new DataBaseProcessingException("Order "
+                    + order + " have problem which product list", e);
```

Comment:
```suggestion
            throw new DataBaseProcessingException("Failed to add products to the order eith id " 
            + orderId, e);
```

Reply:
fixed alert in getOrderProducts() method in OrderDaoJdbcImpl